assets/src/stories-editor/components/context-menu/index.js

```diff
@@ -201,57 +206,58 @@ const ContextMenu = ( props ) => {
 			},
 
 		];
+	}
+
+	// Disable Duplicate Block option for cta and attachment blocks.
+	if ( block && ! DISABLE_DUPLICATE_BLOCKS.includes( block.name ) ) {
+		const dupTitle = ( ! isPage ) ? __( 'Duplicate Block', 'amp' ) : __( 'Duplicate Page', 'amp' );
+		blockActions.push(
+			{
+				name: dupTitle,
+				blockAction: duplicateBlock,
+				params: [ firstBlockClientId ],
+				icon: 'admin-page',
+				className: 'right-click-duplicate',
+			},
+		);
+	}
 
-		if ( block ) {
-			// Disable Duplicate Block option for cta and attachment blocks.
-			if ( ! DISABLE_DUPLICATE_BLOCKS.includes( block.name ) ) {
+	// If more than one page, add options to move blocks between pages.
+	if ( block && ! isPage && numPages > 1 ) {
```

Comment:
Extract `numPages > 1` to a boolean variable, it's used multiple times.

Reply:
I thought about that I was refactoring the code. I will make the change.